fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/JunoOnboardingPageType.kt

```diff
@@ -6,9 +6,31 @@ package org.mozilla.fenix.onboarding.view
 
 /**
  * Model for different types of Onboarding Pages.
+ * @param sequencePosition Position of the page in the onboarding flow, used in telemetry.
+ * @param id Identifier for the page, used in telemetry.
  */
-enum class JunoOnboardingPageType {
-    DEFAULT_BROWSER,
-    SYNC_SIGN_IN,
-    NOTIFICATION_PERMISSION,
+enum class JunoOnboardingPageType(
+    val sequencePosition: String,
+    val id: String,
+) {
+    DEFAULT_BROWSER(
+        sequencePosition = "1",
+        id = "default",
+    ),
+    SYNC_SIGN_IN(
+        sequencePosition = "2",
+        id = "sync",
+    ),
+    NOTIFICATION_PERMISSION(
+        sequencePosition = "3",
+        id = "notification",
+    ),
 }
+
+/**
+ * Helper function for telemetry that maps List<JunoOnboardingPageType> to a string of page names
+ * separated by an underscore.
+ * e.g. [DEFAULT_BROWSER, SYNC_SIGN_IN] will be mapped to "default_sync".
+ */
+fun List<JunoOnboardingPageType>.sequenceId(): String =
```

Comment:
Perhaps
```suggestion
fun List<JunoOnboardingPageType>.pageTypeToTelemetryFormat(): String =
```

Reply:
yeah good point, it's not clear from the name where/how the sequenceId is used, How about just `telemetrySequenceId()`?